index.bs

```diff
@@ -2100,6 +2106,113 @@ Dictionary {{AudioTimestamp}} Members</h5>
+{{AudioRenderPerformance}}</h4>
+
+<pre class="idl">
+[Exposed=Window]
+interface AudioRenderPerformance {
+	undefined start(AudioRenderPerformanceOptions options);
+  	undefined stop();
+  	attribute EventHandler onupdate;
+};
+</pre>
+
+<h5 id="AudioRenderPerformance-attributes">
+Attributes</h5>
+
+<dl dfn-type=attribute dfn-for="AudioRenderPerformance">
+	: <dfn>onupdate</dfn>
+	::
+		An EventHandler for AudioRenderPerformanceEvent.
+</dl>
+
+<h5 id="AudioRenderPerformance-methods">
+Methods</h5>
+
+<dl dfn-type=method dfn-for="AudioRenderPerformance">
+	: <dfn>start(options)</dfn>
+	::
+		Starts performance-related data collection and analysis. This
+		will repeatedly dispatch an {{AudioRenderPerformanceEvent}} to
+		{{AudioRenderPerformance/onupdate}} EventHandler with the given
+		interval in {{AudioRenderPerformanceOptions}}.
+	
+	: <dfn>stop()</dfn>
+	::
+		Stops performance-related data collection and analysis. It also
+		stops the dispatching of {{AudioRenderPerformanceEvent}}.
+</dl>
+
+<h4 dictionary lt="audiorenderperformanceoptions" id="AudioRenderPerformanceOptions">
+{{AudioRenderPerformanceOptions}}</h4>
+
+The {{AudioRenderPerformanceOptions}} dictionary can be used to provide user
+options for an {{AudioRenderPerformance}}.
+
+<pre class="idl">
+dictionary AudioRenderPerformanceOptions {
+  	double updateInterval = 1;
+};
+</pre>
+
+<h5 id="dictionary-AudioRenderPerformanceOptions-members">
+Dictionary {{AudioRenderPerformanceOptions}} Members</h5>
+
+<dl dfn-type=dict-member dfn-for="AudioRenderPerformanceOptions">
+	: <dfn>updateInterval</dfn>
+	::
+		An interval (in second) that capacity/underrun values are
+		updated. A render capacity value is calculated per render
+		quantum, and each value will be collected over the specified
+		interval period. For example, if the renderer runs at a 48Khz
+		sample rate with 128 frames of render quantum over 1 second
+		interval, 375 capacity values will be collected.
+</dl>
+
+<h4 interface lt="audiorenderperformanceevent" id="AudioRenderPerformanceEvent">
+{{AudioRenderPerformanceEvent}}</h4>
+
+<pre class="idl">
+[Exposed=Window]
+interface AudioRenderPerformanceEvent {
+	constructor (DOMString type, double timestamp, double averageCapacity,
+	             double maxCapacity, double underrunRatio);
+  	readonly attribute double timestamp;
+  	readonly attribute double averageCapacity;
+  	readonly attribute double maxCapacity;
+  	readonly attribute double underrunRatio;
+};
+</pre>
+
+<h5 id="AudioRenderPerformanceEvent-attributes">
+Attributes</h5>
+
+<dl dfn-type=attribute dfn-for="AudioRenderPerformanceEvent">
+	: <dfn>timestamp</dfn>
+	::
+		{{BaseAudioContext/currentTime}} of {{AudioContext}} when the
+		event was fired.
+	: <dfn>averageCapacity</dfn>
+	::
+		An average of the collected raw render capacity values over the
+		specified interval. The precision is limited to 1/100th.
+	: <dfn>maxCapacity</dfn>
```

Comment:
Similarly, `maxLoad`. We might be able to find a better name because `maxLoad` could be the maximum acceptable load or something. I'm not sure.

Reply:
Yes. The term "load" is appropriate for the properties.

Reply:
How about "peakLoad"?